Source/ARTPresenceMap.m

```diff
@@ -39,7 +39,7 @@ @interface ARTPresenceMap () {
     ARTPresenceSyncState _syncState;
     ARTEventEmitter<ARTEvent * /*ARTSyncState*/, id> *_syncEventEmitter;
     NSMutableDictionary<NSString *, ARTPresenceMessage *> *_members;
-    NSMutableSet<ARTPresenceMessage *> *_localMembers;
+    NSMutableDictionary<NSString *, ARTPresenceMessage *> *_localMembers; // RTP17h
```

Comment:
Seems outside the scope of this PR to me; introduces a lot of noise in what's already a huge PR (and since we're probably going to end up squashing all of the commits in this PR before merging, given that it has ~100 commits, it doesn't really help to put the change in a separate commit). Can we revert this and create an issue for it?

Reply:
https://github.com/ably/ably-cocoa/issues/1849

Reply:
Sure